### DISCUSSION TITLE
Save Custom made theme with background image

### DISCUSSION THREAD

**Discussion by TanishqSharma2022** [monkeytypegame/monkeytype#5319] [Ideas]
I recently made a custom theme. What I want is to save these themes with background images somewhere and can switch between them whenever I like.
<img width="1440" alt="Screenshot 2024-04-18 at 1 01 26 AM" src="https://github.com/monkeytypegame/monkeytype/assets/123730739/153e43db-aba6-4c3c-84ad-032f365815c3">

**Comment by Warrobot10** [comment]
Hello there! If you go into settings, and scroll down to theme, you can see the preset themes that are already made. You want to click custom, and then save your theme as new. You should then be able to name you theme by hovering over the "custom" name and clicking the pencil.
![image](https://github.com/monkeytypegame/monkeytype/assets/132307750/c440188d-7b74-4fa4-b2a7-9e2b5bd5c7e5)
![image](https://github.com/monkeytypegame/monkeytype/assets/132307750/f2988086-5177-4f0f-9a24-62d92e1a4de5)

**Reply by ImSammy6** [reply]
> Hello there! If you go into settings, and scroll down to theme, you can see the preset themes that are already made. You want to click custom, and then save your theme as new. You should then be able to name you theme by hovering over the "custom" name and clicking the pencil. ![image](https://private-user-images.githubusercontent.com/132307750/333909513-c440188d-7b74-4fa4-b2a7-9e2b5bd5c7e5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTAyNzIsIm5iZiI6MTcxNjk0OTk3MiwicGF0aCI6Ii8xMzIzMDc3NTAvMzMzOTA5NTEzLWM0NDAxODhkLTdiNzQtNGZhNC1iMmE3LTllMmI1YmQ1YzdlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQwMjMyNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMjk3MWJkMzY0ZDZmNWMxMzdiMDdkNThlMDZiMzBkMDg3ZWRlNDNkZTdkNDQwZTI3MmI0NTZhYTI1NTczOTgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YU5xajtIzuesuImIwvbqmNOl6D9_is94pUqRN021CBs) ![image](https://private-user-images.githubusercontent.com/132307750/333909540-f2988086-5177-4f0f-9a24-62d92e1a4de5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NTAyNzIsIm5iZiI6MTcxNjk0OTk3MiwicGF0aCI6Ii8xMzIzMDc3NTAvMzMzOTA5NTQwLWYyOTg4MDg2LTUxNzctNGYwZi05YTI0LTYyZDkyZTFhNGRlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOVQwMjMyNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NTNiNjc1Nzk4ZmNmYTU0Mzc3OThhYWI2MWY5NGJhYjdlZmI1ZjViMzQxODJhZDczYjhlNTlmOWQ0OGFhYTQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ZFmYPOHPPrcIYxjzyL86qMTlchR7XxLc2yvw4Y-yJX4)

I've been wanting the same thing. The issue with this is that it doesn't save the background image with the preset, so if you switch to a different one the background will stay the same
